managed_vms/django_tutorial/README.md

```diff
@@ -0,0 +1,103 @@
+# Getting started with Django on Google Cloud Platform
+
+This repository is an example of how to run a [Django](https://www.djangoproject.com/) 
+app on Google Managed VMs. It uses the [Writing your first Django app](https://docs.djangoproject.com/en/1.9/intro/tutorial01/) as the example app to deploy.
+
+
+## Setup the database
+
+This tutorial assumes you are setting up Django using a SQL database, which is the easiest way to run Django. If you have an existing SQL database running, you can use that, but if not, these are the instructions for creating a managed MySQL instance using CloudSQL.
+
+
+* Create a [CloudSQL instance](https://console.cloud.google.com/project/_/sql/create)
```

Comment:
The creation of instance and allowed networks can be done through gcloud, which I added as an option. The creation of the database and user is currently available in the API but not implemented in the gcloud tool, so instead I mention you can alternatively do the steps using your MySQL client.